src/utils/utils.ts

```diff
@@ -32,3 +32,8 @@ export async function urlToBase64(url: string) {
   const type = url.substring(url.lastIndexOf(".") + 1);
   return `data:image/${type};base64,${imageStr}`;
 }
+
+/** Allows easy way to add a prop to a base object when needing to use complicated getters solution. */
```

Comment:
Returns an object whose keys represent the names of properties to be defined or modified and whose values are objects describing those properties.

Better description. ^